### PR TITLE
Use ::class in test mocks of dav app

### PR DESCRIPTION
**Pull request by MorrisJobke**
Followup to #6934

This time a bit shorter and based on the stuff in the dav app.

### CODE REVIEW

**Review by ChristophWurst — APPROVED** [review]
Looks good

**Comment by rullzer** [comment]
CI is not happy

**Comment by MorrisJobke** [comment]
@nickvergessen @rullzer I found the issue and fixed it. It was indeed a wrong import. But has nothing to do with that error message :/

**Comment by MorrisJobke** [comment]
@nickvergessen @rullzer CI seems to be happy now :)

**Review by nickvergessen — APPROVED** [review]

File: apps/dav/tests/unit/Connector/Sabre/CustomPropertiesBackendTest.php [hunk 1]
```diff
@@ -31,8 +31,9 @@
  * See the COPYING-README file.
  */
 
+use OCA\DAV\Connector\Sabre\Directory;
+use OCA\DAV\Connector\Sabre\File;
 use OCP\IUser;
-use Sabre\DAV\File;
```
Comment (nickvergessen) [inline apps/dav/tests/unit/Connector/Sabre/CustomPropertiesBackendTest.php]: This was used in 217?

Comment (MorrisJobke) [reply apps/dav/tests/unit/Connector/Sabre/CustomPropertiesBackendTest.php]: Yes - but wrongly used. Was a mistake from #6934 🙈

Comment (MorrisJobke) [reply apps/dav/tests/unit/Connector/Sabre/CustomPropertiesBackendTest.php]: https://github.com/nextcloud/server/pull/6934/files#diff-e4a64f1ed7ba4acbd53c73bbd395a5fbL212

**Review by rullzer — APPROVED** [review]

**Pull request merged**